.travis.yml

```
@@ -29,7 +29,9 @@ matrix:
     - php: 7.3
       env: WP_VERSION=5.2 WP_MULTISITE=1
     - php: 7.3
-      env: WP_VERSION=latest WP_MULTISITE=1
+      env:
+        env: WP_VERSION=latest WP_MULTISITE=1 PHPCS=1
+        - PHPCS=1
```

```suggestion
      env: WP_VERSION=latest WP_MULTISITE=1 PHPCS=1
```

:crossed_fingers: that this works better.

---

Great! Thank you, @jrfnl.

---

@rfmeier I see a merge, but this change hasn't been included and the Travis script is effectively broken without it... did I miss something ?